### PR TITLE
Status banner

### PR DESCRIPTION
**Pull request by hetd54**
As a reminder, we are using App Hosting now, so no previews. Here's a preview from local -- 

When there are open issues:
Screenshot 2024-12-12 at 9 26 23 AM

When no issues:
Screenshot 2024-12-12 at 9 25 21 AM

We should decide what colors would look nice in the banner -- please give suggestions!

### CODE REVIEW

File: components/header/utils.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        org: { org },
+        repo: { name },
+        type: "private",
+        sort: "created",
+        direction: "desc",
+      })
+
+      // This currently only gets the last 100 comments on the repo. It should be changed
+      // to get the comments per issue instead, which will be addressed in the next PR
+      const comments = await octokit.request(
```
Comment (s-bessey) [inline components/header/utils.ts]: No need to get the issue comments here!

File: components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import Banner from "@/components/header/Banner"
+import Navbar from "@/components/header/Navbar"
+import getOpenIssues from "@/components/header/utils"
+
+export default async function Header() {
+  const repositories = await getOpenIssues()
+  // uncomment to test banner with no open issues
```
Comment (s-bessey) [inline components/header/Header.tsx]: I'm a little confused about the comment vs uncomment for testing here? Might be good to make that a little clearer

**Review by s-bessey — CHANGES_REQUESTED** [review]
A couple of nitpicks!

**Comment by hetd54** [comment]
Gradient with green and blue colors?
Screenshot 2024-12-05 at 6 12 22 PM
Screenshot 2024-12-05 at 6 22 46 PM

**Comment by eldu** [comment]
@hetd54 @s-bessey 
I think we'll want to streamline the language we're using. It seems like we're using issues/incidents interchangeably. Let's keep it consistent with the status page and the website?

Also,
When all services are operational "View Issues" doesn't seem to make sense since issues could imply something is wrong.
Maybe instead of "View Issues", "View Status"?

**Comment by s-bessey** [comment]
I think the current color blends a bit too much with the rest of the page -- I'd say something a bit darker and without _too_ much contrast between the "top" and "bottom" of the gradient

File: components/header/Banner.tsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ interface BannerProps {
   variant: string
 }
 
+// Try red/orange for the error gradient
```
Comment (s-bessey) [inline components/header/Banner.tsx]: remove comment?

File: components/header/Header.tsx [hunk 4]
```diff
@@ -1,66 +1,52 @@
 import Banner from "@/components/header/Banner"
 import Navbar from "@/components/header/Navbar"
 import getOpenIssues from "@/components/header/utils"
+import { ExclamationTriangleIcon } from "@heroicons/react/24/outline"
 
+// ONLY HOME PAGE!
+
+// Green/orange dot?? See if that feels like recording
```
Comment (s-bessey) [inline components/header/Header.tsx]: Still some notes to self?

**Review by s-bessey — APPROVED** [review]
Looking good!

**Pull request merged**